src/corner_kick/src/containers/mainUI/GlobalStyle.tsx

```diff
@@ -0,0 +1,186 @@
+        grid-row: 1 / 4;
+        grid-column: 1;
+        border-top: 1px solid ${(props) => props.theme.colors.border};
+        border-right: 1px solid ${(props) => props.theme.colors.border};
+
+        display: flex;
+        flex-flow: column nowrap;
+
+        overflow: hidden;
+    }
+
+    #sidebar {
+        grid-row: 2 / 4;
+        grid-column: 2;
+        border-top: 1px solid ${(props) => props.theme.colors.border};
+        border-right: 1px solid ${(props) => props.theme.colors.border};
+        overflow: hidden;
+    }
+
+    #mainTitle {
+        grid-row: 1;
+        grid-column: 3;
+
+        display: flex;
+
+        overflow: hidden;
+    }
+
+    #main {
+        grid-row: 2;
+        grid-column: 3;
+        background: ${(props) => props.theme.colors.panel};
+        overflow: hidden;
+        border-top: 1px solid ${(props) => props.theme.colors.border};
+    }
+
+    #console {
+        grid-row: 3;
+        grid-column: 3;
+        overflow: hidden;
+        border-top: 1px solid ${(props) => props.theme.colors.border};
+    }
+
+    #footerLeft {
+        grid-row: 4;
+        grid-column: 1 / 3;
+        border-top: 1px solid ${(props) => props.theme.colors.border};
+
+        display: flex;
+        justify-content: flex-start;
+        align-items: center;
+        padding: 0px 4px;
+        overflow: hidden;
+    }
+
+    #footerRight {
+        grid-row: 4;
+        grid-column: 3;
+        border-top: 1px solid ${(props) => props.theme.colors.border};
+
+        display: flex;
+        justify-content: flex-end;
+        align-items: center;
+        padding: 0px 4px;
+        overflow: hidden;
+    }
+`;
+
+const mapStateToProps = (state: IRootState) => ({
+    settings: state.settings,
+});
+
+interface ISettingsMainProps {
+    settings: ISettingsState;
+}
+
+class GlobalStyleInternal extends React.Component<ISettingsMainProps> {
+    public render() {
+        const { settings } = this.props;
+
+        // tslint:disable:object-literal-sort-keys
+        const theme: IThemeProvider = {
+            colors: {
+                fg: settings.colors_fg,
+                bg: settings.colors_bg,
+
+                accent: settings.colors_accent,
+                subdued: settings.colors_subdued,
+                selected: settings.colors_selection,
+
+                panel: settings.colors_panel,
+                border: settings.colors_border,
+
+                success: settings.colors_success,
+                error: settings.colors_error,
+                warn: settings.colors_warn,
+                info: settings.colors_info,
+                debug: settings.colors_debug,
+
+                red: settings.colors_red,
+                orange: settings.colors_orange,
+                yellow: settings.colors_yellow,
+                green: settings.colors_green,
+                cyan: settings.colors_cyan,
+                blue: settings.colors_blue,
+                purple: settings.colors_purple,
```

Comment:
This may be a question for when we do layering, but if I wanted to add a new color, how would I know everywhere to look? These seem somewhat scattered right now.

Reply:
That's fair, let's revisit that when we do layering. I added that requirement to #201